src/components/Pages/Processes.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { withRouteData } from 'react-static';
+import { injectIntl } from 'react-intl';
+
+import { processes as i18n } from 'js/i18n/definitions';
+
+import PageHeader from 'components/PageHeader';
+import TileGroup from 'components/Tiles/TileGroup';
+
+const Processes = ({ processes, intl }) => (
+  <div>
+    <div className="wrapper wrapper--sm container-fluid">
+      <PageHeader
+        title="Use City of Austin Processes"
+        description="The City of Austin provides hundreds of services to people. This is a short list of processes that will grow over time."
```

Comment:
more i18n 🌍

Reply:
This is generic text I made up that I copy-pasta-ed from services just so we could have an easy place to test process pages. so when we have a real processes page (if we do), I can update this.